modules/dmrpp_module/DmrppCommon.h

```diff
@@ -61,6 +61,24 @@ class SuperChunk;
 
 void join_threads(pthread_t threads[], unsigned int num_threads);
 
+/**
+ * @brief hold the value used to fill empty chunks
+ */
+union fill_value_union {
```

Comment:
Sweet! It's been a long time since I've seen/used a `union`.

Reply:
Yeah.